app/serializers/articles_show_serializer.rb

```diff
@@ -0,0 +1,14 @@
+class ArticlesShowSerializer < ActiveModel::Serializer
+  attributes :id, :title, :body, :date, :author
+
+  def author
+    {
+      first_name: object.user.first_name,
+      last_name: object.user.last_name
+    }
```

Comment:
Okay, I'll buy this. But it does not come across in the user story

Reply:
Super, user story is updated as well to accommodate

Reply:
For the future, the stories are locked down once voted on and should stay as they are. If they need to change - then the process should be redone. Please bring this up in your next team retrospective.

Reply:
We added it as a task before voting and it was meant to be included in "details" in the user story - but I see how this is not coming across clearly

Reply:
okay